actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/UserBulkMigrationActor.java

```diff
@@ -48,7 +49,6 @@
     private Util.DbInfo usrDbInfo = Util.dbInfoMap.get(JsonKey.USER_DB);
     private static ObjectMapper mapper=new ObjectMapper();
     private static SystemSetting systemSetting;
-    private OrganisationClient organisationClient = new OrganisationClientImpl();
     @Override
     public void onReceive(Request request) throws Throwable {
         Util.initializeContext(request, TelemetryEnvKey.USER);
```

Comment:
In case of bulk user upload , context object won't work. once response is completed , context object will be invalidated for that thread.